Detections/ThreatIntelligenceIndicator/IPEntity_AzureKeyVault.yaml

```diff
@@ -0,0 +1,51 @@
+id: 57c7e832-64eb-411f-8928-4133f01f4a25
+name: TI map IP entity to Azure Key Vault logs
+description: |
+  'Identifies a match in Azure Key Vault logsfrom any IP IOC from TI'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: ThreatIntelligence
```

Comment:
We should add the connector for Azure Key Vault Logs here as well, or do we not have one?